src/Aquifer.Public.API/Endpoints/Bibles/List/Endpoint.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Aquifer.Data;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.List;
+
+public class Endpoint(AquiferDbContext dbContext) : Endpoint<Request, IReadOnlyList<Response>>
+{
+    private readonly AquiferDbContext _dbContext = dbContext;
+
+    public override void Configure()
+    {
+        Get("/bibles");
+        Description(d => d.ProducesProblemFE());
+        Summary(s =>
+        {
+            s.Summary = "Get a list of Bibles.";
+            s.Description =
+                "For a given optional language ID, returns the Bibles in the system including language information, abbreviation codes, whether or not the Bible supports a Greek alignment, and license information.";
+        });
+    }
+
+    public override async Task HandleAsync(Request request, CancellationToken ct)
+    {
+        var bibles = await _dbContext.Bibles
+            .Where(b =>
+                b.Enabled &&
+                !b.RestrictedLicense &&
+                (!request.LanguageId.HasValue || b.LanguageId == request.LanguageId) &&
+                (request.LanguageCode == null || b.Language.ISO6393Code == request.LanguageCode) &&
+                (!request.IsLanguageDefault.HasValue || b.LanguageDefault == request.IsLanguageDefault) &&
+                (!request.HasGreekAlignment.HasValue || b.GreekAlignment == request.HasGreekAlignment) &&
+                (!request.HasAudio.HasValue || b.BibleBookContents.Any(bbc => bbc.AudioUrls != null) == request.HasAudio))
+            .Select(bible => new Response
+            {
+                Name = bible.Name,
+                Abbreviation = bible.Abbreviation,
+                Id = bible.Id,
+                SerializedLicenseInfo = bible.LicenseInfo,
+                LanguageId = bible.LanguageId,
+                IsLanguageDefault = bible.LanguageDefault,
+                HasGreekAlignment = bible.GreekAlignment,
```

Comment:
Eh, let's comment this out until we actually add Greek Alignment to the API. If this is there, someone's gonna complain that they can't get it.

Reply:
Good call.  I should have confirmed it was available on the public API before copying the internal API filtering.

Reply:
[Commit](https://github.com/BiblioNexusStudio/aquifer-server/pull/443/commits/144e1c77cd0e32015d7103ac7385c5811a2153bb) to remove `HasGreekAlignment` filtering.